src/actions.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { isNgrxFormsAction } from './actions';
+
+describe('action', () => {
+  it('should return true if type is ngrx/forms/', () => {
+    expect(isNgrxFormsAction({ type: 'ngrx/forms/' })).toBeTruthy();
+  });
```

Comment:
Please try to be consistent with the formatting of the rest of the codebase, i.e. leave a blank line between each test.